netdump.c

```diff
@@ -1927,6 +1927,12 @@ vmcoreinfo_read_string(const char *key)
 			pc->read_vmcoreinfo = no_vmcoreinfo;
 			return value;
 		}
+		if (STREQ(key, "NUMBER(tcr_el1_t1sz)") && nd->arch_data2) {
+			value = calloc(VADDR_PRLEN+1, sizeof(char));
+			sprintf(value, "%lld", ((ulonglong)nd->arch_data2 >> 32) & 0xffffffff);
+			pc->read_vmcoreinfo = no_vmcoreinfo;
+			return value;
+		}
```

Comment:
Is this part needed?

This checks the `key` argument of `vmcoreinfo_read_string()` and the only caller with that argument is `arm64_calc_VA_BITS()` above.  Once `pc->read_vmcoreinfo("NUMBER(TCR_EL1_T1SZ)")` is called, the `pc->read_vmcoreinfo` is set to `no_vmcoreinfo`, so it looks like this part is not used.

Reply:
I didn't need that changed for my local testing  . I would suggest adding that too .

Reply:
> I didn't need that changed for my local testing . I would suggest adding that too .

Why?

I don't think this block is needed, but if you suggest adding the lowercase block too, how about this? to avoid confusion about lack of lowercase name here.
```
--- a/netdump.c
+++ b/netdump.c
@@ -1921,7 +1921,8 @@ vmcoreinfo_read_string(const char *key)
                        sprintf(value, "%ld", nd->arch_data2 & 0xffffffff);
                        return value;
                }
-               if (STREQ(key, "NUMBER(TCR_EL1_T1SZ)") && nd->arch_data2) {
+               if ((STREQ(key, "NUMBER(TCR_EL1_T1SZ)") ||
+                   STREQ(key, "NUMBER(tcr_el1_t1sz)")) && nd->arch_data2) {
                        value = calloc(VADDR_PRLEN+1, sizeof(char));
                        sprintf(value, "%lld", ((ulonglong)nd->arch_data2 >> 32) & 0xffffffff);
                        pc->read_vmcoreinfo = no_vmcoreinfo;
```

Reply:
Hi,

I have no problem with any refactoring you feel is necessary. 

JD.

Reply:
Ok, will proceed with this.

Thanks,
Kazu